lib/internal/errors.js

```diff
@@ -154,12 +157,21 @@ E('ERR_UNKNOWN_BUILTIN_MODULE', (id) => `No such built-in module: ${id}`);
 E('ERR_UNKNOWN_SIGNAL', (signal) => `Unknown signal: ${signal}`);
 E('ERR_UNKNOWN_STDIN_TYPE', 'Unknown stdin file type');
 E('ERR_UNKNOWN_STREAM_TYPE', 'Unknown stream file type');
+E('ERR_RENEGOTIATE', 'Failed to renegotiate');
+E('ERR_REQD_SERVER_NAME',
+  '"servername" is required parameter for Server.addContext');
 E('ERR_SOCKET_ALREADY_BOUND', 'Socket is already bound');
 E('ERR_SOCKET_BAD_TYPE',
   'Bad socket type specified. Valid types are: udp4, udp6');
 E('ERR_SOCKET_CANNOT_SEND', 'Unable to send data');
 E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536');
+E('ERR_SOCKET_CLOSED', 'Socket is closed');
 E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
+E('ERR_TLS_HANDSHAKE_TIMEOUT', 'TLS handshake timeout');
+E('ERR_TLS_SESSION_ATTACK', 'TSL session renegotiation attack detected');
+E('ERR_TLS_SESSION_CALLBACK', 'TSL sesssion callback was called 2 times');
+E('ERR_TLS_OCSP_CALLBACK', 'TLS OCSP callback was called 2 times');
+E('ERR_TLS_SNI_CALLBACK', 'TLS SNI callback was called 2 times');
```

Comment:
Maybe add "_ALREADY_CALLED" to these 2 as well.  It does make the code long but it also makes it clearer what's going wrong.